fastify.js

```diff
@@ -249,7 +249,10 @@ function build (options) {
   }
 
   function hookIterator (fn, cb) {
-    fn(this.req, this.res, cb)
+    const ret = fn(this.req, this.res, cb)
```

Comment:
Since this is an hot path, use `var` instead of `const`, it has slightly better performances :)
Same in the others iterators.

Reply:
Didn't notice that :)